ui/components/KubeStatusIndicator.tsx

```diff
@@ -12,15 +12,23 @@ type Props = {
   short?: boolean;
   suspended?: boolean;
 };
+export enum ReadyType {
+  Ready = "Ready",
+  NotReady = "NotReady",
```

Comment:
If you need space to display the "Not Ready" text you can keep it in the string value:

```
NotReady = "Not Ready",
```